### PR TITLE
Data Types, Operators and Expressions

### LINKED ISSUE #25: Data Types, Operators and Expressions
1. 交换2个数a和b，分别用引入和不引入第3个变量（多种不同方法）来实现，同时要求以注释的形式描述各方法的优缺点。输出格式："{a} {b} => {} {}"
3. 判断2个浮点数a和b是否相等。输出格式："{a} {b} {dt} eq/ne" (eq和ne分别表示“相等”或“不等”)。
4. 判断某个年份y是否为润年。输出格式："{y} yes/no"
5. 判断一个整数n是否为偶数 (分别用算术运算符和位运算符实现)。输出格式："{n} even/odd"。
6. 判断一个整数n是否为2<sup>k</sup>(2的k次方)。输出格式："{n} yes/no"。
7. n为一整数，不用乘法运算符求n x 17。输出格式："{n} x 17 = {}"。
8. n为一整数，不用取模运算符求n % 8。输出格式："{n} % 8 = {}"。
9.  在一个8位计算机上有个整数n，计算n/16。输出格式："{n} / 16 = {}"。
10. 设Unix-like系统中某文件当前的模式为m，请用位运算分别实现如下4种操作。输出格式 ："{m}: {} {} {} {}"，并且要求全部的4个new m均基于m初始值计算而得。
    - chmod 600
    - chmod +x
    - chmod u+w
    - chmod go-w

Fork #17

### PR DESCRIPTION
**Pull request by Wick-Lau**
Resolve #25

Signed-off-by: Wick Liu <wick.liu@outlook.com>

### CODE REVIEW

**Comment by Wick-Lau** [comment]
No. | Output | Format | Result
--- | ------ | ------ | ------
1 | 11 22 => 22 11 | {} {} => {} {} | True
2 | 11 22 => 22 11 | {} {} => {} {} | True
3 | 11 22 => 22 11 | {} {} => {} {} | True
4 | 11 22 => 22 11 | {} {} => {} {} | True
5 | 3.1415 3.1415 0.0001 eq | {} {} {} {} | True
6 | 2020 yes | {} {} | True
7 | 15 odd | {} {} | True
8 | 15 odd | {} {} | True
9 | 9 no | {} {} | True
10 | 8 x 17 = 136 | {} x 17 = {} | True
11 | 30 % 8 = 6 | {} % 8 = {} | True
12 | 32 / 16 = 2 | {} / 16 = {} | True
13 | 644: 600 755 644 644 | {}: {} {} {} {} | True

File: operators/wicklau.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+print(s + " => {} {}".format(a, b))
+
+a = 3.1415
+b = 3.1415
+c = 0.0001
+if a > b - c and a < b + c:
+    print("{} {} {} {}".format(a, b, c, "eq"))
+else:
+    print("{} {} {} {}".format(a, b, c, "ne"))
+
```
Comment (conke) [inline operators/wicklau.py]: 这种情况不适合用两个print。下同

**Comment by Wick-Lau** [comment]
No. | Output | Format | Result
--- | ------ | ------ | ------
1 | 11 22 => 22 11 | {} {} => {} {} | True
2 | 11 22 => 22 11 | {} {} => {} {} | True
3 | 11 22 => 22 11 | {} {} => {} {} | True
4 | 11 22 => 22 11 | {} {} => {} {} | True
5 | 3.1415 3.1415 0.0001 eq | {} {} {} {} | True
6 | 2020 yes | {} {} | True
7 | 15 odd | {} {} | True
8 | 15 odd | {} {} | True
9 | 9 no | {} {} | True
10 | 8 x 17 = 136 | {} x 17 = {} | True
11 | 30 % 8 = 6 | {} % 8 = {} | True
12 | 32 / 16 = 2 | {} / 16 = {} | True
13 | 644: 600 755 644 644 | {}: {} {} {} {} | True

File: operators/wicklau.py [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+#优点:效率比较高
+#缺点:多分配了中间变量的内存
+a = 11
+b = 22
+s = "{} {}".format(a, b)
+c = a
+a = b
+b = c
+print(s + " => {} {}".format(a, b))
```
Comment (conke) [inline operators/wicklau.py]: why not just: `print(s, '=>', a, b)`?

**Pull request merged**